xrayutilities/materials/material.py

```diff
@@ -464,31 +464,30 @@ class Crystal(Material):
     """
     Crystalline materials are described by this class
     """
-    def __init__(self, name, lat, cij=None, thetaDebye=None):
-        super(Crystal, self).__init__(name, cij)
 
-        self.lattice = lat
+    def __init__(self,name, lattice=None, thetaDebye=None, cij=None):
+        super(Crystal, self).__init__(name, cij=cij)
+
+        if lattice is not None:
+            self.lattice = lattice
+
         if isinstance(thetaDebye, numbers.Number):
             self.thetaDebye = float(thetaDebye)
         else:
             self.thetaDebye = thetaDebye
 
-    @classmethod
-    def fromCIF(cls, ciffilename):
+    # @staticmethod
+    def fromCIF(self, ciffilename):
         """
-        Create a Crystal from a CIF file. Name and
+        Constructs a SGLattice objects from a .cif file and loads it into Crystal.lattice
 
```

Comment:
Can you explain the motivation of these changes?
for me `fromCIF` is a perfect example of a classmethod. I see that you want to load only the lattice from the cif file and keep the other things as defined previously. This might also be a valid use case. Maybe one should supply two functions to do these two different things

Reply:
agreed. 
Two distinct methods should be used. 
Will send pull request with mods once occasion arises.